cfme/scripting/report_parser.py

```diff
@@ -0,0 +1,97 @@
+"""Script to parse test reports."""
+import json
+import smtplib
+from email.MIMEMultipart import MIMEMultipart
```

Comment:
Please update for py3 compatibility: `from email.mime.multipart import MIMEMultipart`

Reply:
Done.